lib/test/cdo/contact_rollups/test_pardot_helpers.rb

```diff
@@ -33,4 +33,22 @@ def test_raise_if_response_error_no_error
 
     assert_nil PardotHelpersTest.send(:raise_if_response_error, pardot_ok)
   end
+
+  def test_try_with_exponential_backoff_one_retry
+    tries = 0
```

Comment:
If I understand correctly this is a way to track the number of tries that actually occurred in this test? Possible to name this something else (eg, `tries_counter`? Got confused looking back and forth to method which also has a counter called `tries`?

Reply:
Also might just be more clear to someone with more experience with blocks/retry/etc.